internal/clients/http/sources/sources_client.go

```diff
@@ -189,11 +189,8 @@ func (c *sourcesClient) GetAuthentication(ctx context.Context, sourceId string)
 	// Sources API currently does not provide a good server-side filtering.
 	auth, err := filterSourceAuthentications(*resp.JSON200.Data)
 	if err != nil {
-		at := make([]string, 0)
-		for _, auth := range *resp.JSON200.Data {
-			at = append(at, string(*auth.Authtype))
```

Comment:
should we keep that?
I assume resp.Body does not give the same info as resp.JSON200.Data

Reply:
Sure, let me slightly change it tho.